open-metadata-implementation/governance-servers/docs/README.md

```diff
@@ -21,7 +21,6 @@ entirely by changes in metadata and the IT landscape.
 | Open Lineage Server | Provides a historic reporting warehouse for lineage. | [Open Lineage Services](../open-lineage-services) | [Asset Lineage OMAS](../../access-services/asset-lineage) | No |
 | Security Officer Server | Provide operational support for the security policies defined by the security officer. | [Security Officer Services](../security-officer-services) | [Security Officer OMAS](../../access-services/security-officer) | No |
```

Comment:
Remove SecurityOfficerServices and SecuritySyncServices?  Looks like the engine host services is missing?

Reply:
I can remove securityofficerservices & securitysyncservices as per this discussion thread.
I've not touched engine host services - I guess you noticed the omission - I can add that in

Reply:
Can you comment with the text you want added for engine host services? In particular the omas pairing? Or we can add back in another issue

Reply:
Modified services removed from this doc.